docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml

```diff
@@ -20,28 +20,53 @@ sections:
 
       - question: What is the timeline for shutting down legacy tokens?
         answer: |
-          After October 2024, Outlook add-ins that rely on legacy Exchange tokens will not work as expected since Exchange will no longer issue those legacy tokens by default.
+          The following tables list the key milestones based on which channel customers are using. Note that the general availability (GA) date for NAA will vary based on channel. We'll provide tooling for administrators to reenable Exchange tokens for tenants and add-ins if those add-ins are not yet migrated to NAA.
```

Comment:
Do we know how that tooling will be provided?

Reply:
That info is coming later...